website/content/docs/upgrade/index.mdx

```diff
@@ -13,8 +13,8 @@ version to the next. Upgrades should cause neither a Nomad nor a service
 outage. However, there are some restrictions to be aware of before upgrading:
 
 - Nomad strives to be backward compatible for at least 1 point release, so
-  Nomad v0.10 hosts work with v0.9 hosts. Upgrading 2 point releases (eg v0.8
-  to v0.10) may work but is untested and unsupported.
+  Nomad v1.6.x hosts work with v1.5.x hosts. Upgrading 2 point releases
+  (eg v1.4.0 to v1.6.0) may work but is untested and unsupported.
```

Comment:
```suggestion
  (for example, from v1.4.0 to v1.6.0) may work but is untested and unsupported.
```

As a non-native speaker, I find `i.e.`, `e.g.`, and other acronyms like these very confusing 😅